public/doc/openapi-3-v1.0.0.json

```diff
@@ -593,6 +593,46 @@
         }
       }
     },
+    "/rbac/seeded": {
```

Comment:
@syncrou I don't think we should have this API. The caller can find that out himself. I think the seeded check is an internal thing for us which allows us to quickly respond to a  seed request when called.